server/index.js

```diff
@@ -98,6 +135,10 @@ const loop = () => {
           tempGrid.setWalkableAt(pl.tileX, pl.tileY, false)
         );
 
+        if (playerNew.follow) {
+          setFollow(playerNew);
```

Comment:
`setFollow` requires `player.follow` to be set, maybe move if to inside `setFollow`?